### PR TITLE
Fix medium tests

### PR DESCRIPTION
**Pull request by LyzardKing**
The medium tests all work now.
The next thing to do is look at the large tests.
Mostly they should work, but I can't get the vm to run properly.

### CODE REVIEW

File: tests/medium/__init__.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ContainerTests(LoggedTestCase):
     """Container-based tests utilities"""
 
     DOCKER_USER = "user"
-    DOCKER_TESTIMAGE = "didrocks/docker-umake-manual"
+    DOCKER_TESTIMAGE = "e3ff8df5bdd8"
```
Comment (didrocks) [inline tests/medium/__init__.py]: Please remove that one (and rebase so that it's not a double change :))

File: umake/frameworks/baseinstaller.py [hunk 2]
```diff
@@ -137,6 +136,9 @@ def remove(self):
     def confirm_path(self, path_dir=""):
         """Confirm path dir"""
 
+        if self.desktop_filename:
```
Comment (didrocks) [inline umake/frameworks/baseinstaller.py]: Why did you move that piece, you didn't comment it?

File: umake/frameworks/ide.py [hunk 3]
```diff
@@ -77,7 +77,6 @@ def __init__(self, *args, **kwargs):
             kwargs["required_files_path"] = current_required_files_path
         download_page = 'https://www.eclipse.org/downloads/'
         kwargs["download_page"] = download_page
-        kwargs["exec_rel_path"] = "eclipse"
```
Comment (didrocks) [inline umake/frameworks/ide.py]: I still find it being mentioned twice in baseinstaller.py:
```umake/frameworks/baseinstaller.py:        self.exec_rel_path = kwargs.get("exec_rel_path", None)
umake/frameworks/baseinstaller.py:                          "match_last_link", "exec_rel_path"]:
```

Did you remove them and I didn't notice it? If we are going to remove all of them, better to ensure it's not anywhere anymore :)

**Comment by didrocks** [comment]
Here we go! Just some comments and questions to be answered before merging. Otherwise, good job! :)

File: tests/medium/__init__.py (hunk 1 above)
Comment (LyzardKing) [inline tests/medium/__init__.py]: Ah yes... This was not supposed to be pushed. It was just to run the tests.

File: umake/frameworks/baseinstaller.py (hunk 2 above)
Comment (LyzardKing) [inline umake/frameworks/baseinstaller.py]: I moved this from the end of confirm_path to the start.
The medium baseInstaller test was failing.
The function hit a return before it could set the exec_path

File: umake/frameworks/ide.py (hunk 3 above)
Comment (LyzardKing) [inline umake/frameworks/ide.py]: It should be removed..
I made the changes on the wrong branch, so it might be that I forgot some when creating a new one

**Comment by didrocks** [comment]
Merging this one as well now!
